.travis.yml

```diff
@@ -32,13 +32,19 @@ jobs:
   - stage: deploy
     script: "./gradlew publish -PnexusBaseUrl=$nexusBaseUrl -PnexusUpRepo=$nexusUpRepo
       -PnexusUser=$nexusUser -PnexusPassword=$nexusPassword"
+addons:
+  sonarcloud:
+    organization: "starlibs" # the key of the org you chose at step #3
+script:
+  # other script steps might be done before running the actual analysis
+  - sonar-scanner
```

Comment:
I doubt that this can work. Besides, in my view, the Travis-Log does **not** say that SonarQube was called but only that the addon was registered. We have a stages travis build here, so I would suggest to add the script entry to the build stage, e.g., before or after the JavaDoc entry.

Reply:
Like said, this is still WIP.